### ISSUE TITLE
Upstream Things we need from TutorialKit

### ISSUE THREAD

**Issue by NullVoxPopuli** [NullVoxPopuli/warp-drive-tutorial#1]
I can PR fixes here, waiting on guidance
- https://github.com/stackblitz/tutorialkit/issues/400

Blockers:
- https://github.com/stackblitz/tutorialkit/issues/401

Nice to haves:
- https://github.com/stackblitz/tutorialkit/issues/213
